src/modding/ModManager.cs

```diff
@@ -880,11 +1576,14 @@ private void OpenModInfoPopup()
         fullInfoAuthor.Text = info.Author;
         fullInfoVersion.Text = info.Version;
         fullInfoDescription.Text = info.Description;
-        fullInfoLongDescription.Text = info.LongDescription;
+        fullInfoLongDescription.ExtendedBbcode = info.LongDescription;
         fullInfoFromWorkshop.Text = selectedMod!.Workshop ?
             TranslationServer.Translate("THIS_IS_WORKSHOP_MOD") :
             TranslationServer.Translate("THIS_IS_LOCAL_MOD");
         fullInfoIconFile.Text = info.Icon;
+        string fullInfoPreviewImagesText = string.Empty;
+        info.PreviewImages?.ForEach(s => fullInfoPreviewImagesText += "* " + s + "\n");
```

Comment:
A bunch of places in this method that also need string formatting. Also I'd make this "FormatAsList" operation a helper method that return empty string with null or empty list and otherwise does the string build. Also this should use string builder or probably more likely string.Join as this seems simpler to implement that way.